requirements.txt

```diff
@@ -1 +1 @@
-Theano==0.7
+git+https://github.com/Theano/Theano.git@4dacb9e7d04d842d8ffe2c948dbc90722ae990d4#egg=Theano
```

Comment:
Ok, didn't realize that. The reason I'm pushing for this because one of the examples I made for recipes is broken without this feature.